src/nets/me2e_asr_attctc.py

```diff
@@ -0,0 +1,204 @@
+#!/usr/bin/env python
```

Comment:
better to rename it as `e2e_asr_mc_attctc.py`

Reply:
I see.
I will change it later.